snow/snowtest/snowtest.go

```diff
@@ -35,9 +38,28 @@ func (noOpAcceptor) Accept(*snow.ConsensusContext, ids.ID, []byte) error {
 	return nil
 }
 
+func EmptyContext() *snow.Context {
+	sk, err := bls.NewSecretKey()
+	if err != nil {
+		panic(err)
+	}
+	pk := bls.PublicFromSecretKey(sk)
+	return &snow.Context{
+		NetworkID:    0,
+		SubnetID:     ids.Empty,
+		ChainID:      ids.Empty,
+		NodeID:       ids.EmptyNodeID,
+		PublicKey:    pk,
+		Log:          logging.NoLog{},
+		BCLookup:     ids.NewAliaser(),
+		Metrics:      metrics.NewOptionalGatherer(),
+		ChainDataDir: "",
+	}
+}
```

Comment:
I know this is what we had before... but I still hate this... Are there places that we currently use `EmptyContext` that we could just be using `Context`?

Reply:
Removed it altogether